packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -116,10 +137,200 @@ export default class VTable {
                     cell.td.style.borderBottom = getBorderStyle(format.bottomBorderColor);
                     cell.td.style.borderLeft = getBorderStyle(format.verticalBorderColor);
                     cell.td.style.borderRight = getBorderStyle(format.verticalBorderColor);
+                    cell.td.style.borderBottom = getBorderStyle(format.bottomBorderColor);
+                    cell.td.style.backgroundColor = format.bgColor;
                 })
         );
     }
 
+    /**
+     * Set color to even and odd rows
+     * @param format
+     */
+    private setTableRowColor(format: Partial<TableFormat>) {
+        if (!format.bandedRows) {
+            return;
+        }
+        this.cells.forEach((row, index) => {
+            if (index % 2 === 0) {
+                row.forEach(cell =>
+                    cell.td && format.bgColorEven
+                        ? (cell.td.style.backgroundColor = format.bgColorEven)
+                        : ''
+                );
+            } else {
+                row.forEach(cell =>
+                    cell.td && format.bgColorOdd
+                        ? (cell.td.style.backgroundColor = format.bgColorOdd)
+                        : ''
+                );
+            }
+        });
+    }
+
+    /**
+     * Set color to even and odd columns
+     * @param format
+     */
+    private setTableColumnsColor(format: Partial<TableFormat>) {
+        if (!format.bandedColumns) {
+            return;
+        }
+
+        this.cells.forEach(row => {
+            row.forEach((cell, index) => {
+                if (index % 2 === 0 && cell.td && format.bgColumnColorEven) {
+                    cell.td.style.backgroundColor = format.bgColumnColorEven;
+                } else if (index % 2 === 1 && cell.td && format.bgColumnColorOdd) {
+                    cell.td.style.backgroundColor = format.bgColumnColorOdd;
+                }
+            });
+        });
+    }
+
+    /**
+     * Organize the borders of table
+     * @param format
+     * @returns
+     */
+    private setBorderType(format: Partial<TableFormat>) {
+        if (!format.tableBorderFormat) {

```

Comment:
Maybe just merge this check into the switch block below:

switch (tableborderformat) {
  ... 
  case TableBorderFormat.None;
    break; // no no
}

Then you can add a "None" value of TableBorderFormat